### PR TITLE
New package: fann-2.2.0

### PR DESCRIPTION
**Pull request by diogoleal**

### CODE REVIEW

**Comment by xtraeme** [comment]
Shared libraries without a development packages is not going to be useful.

**Comment by diogoleal** [comment]
Sorry!
development package added.

**Comment by xtraeme** [comment]
Incomplete... headers should be moved to the development package.

**Comment by diogoleal** [comment]
done!

File: srcpkgs/fann/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+license="LGPL-2.1"
+homepage="http://leenissen.dk/fann/"
+distfiles="https://github.com/libfann/fann/archive/${version}.tar.gz"
+checksum=f31c92c1589996f97d855939b37293478ac03d24b4e1c08ff21e0bd093449c3c
+
+fann-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include/*.h
```
Comment (xtraeme) [inline srcpkgs/fann/template]: Nope... just use `vmove usr/include`

**Pull request merged**